src/pages/Game/index.js

```diff
@@ -24,40 +29,46 @@ const Game = () => {
        
         setStepNumber(step)
         setXIsNext((step % 2) === 0)
+       
         
       }
-     
+    
     const current = history[stepNumber];
-    const winner = calculateWinner(current.squares);
-    const moves = history.map((step, move) => {
+    const line = calculateWinner(current.squares);
+    const moves = history.map((step, move) => { 
       const desc = move ?
-        'Go to move #' + move :
+        'Go to move #' + move + '  Last move  C' +((movment[move-1]%3)+1) + 'F' +((Math.floor(movment[move-1]/3))+1)  :
```

Comment:
Is clearer if you move complicated calculations to their own constants with clear names.

`const column = (movment[move-1]%3)+1`
`const row = (Math.floor(movment[move-1]/3))+1`

```suggestion
        'Go to move #' + move + '  Last move  C' +(column) + 'F' +(row)  :
```